csclient/csclient_test.go

```diff
@@ -84,11 +86,21 @@ func (s *suite) startServer(c *gc.C, session *mgo.Session) {
 		return s.discharge(cond, arg)
 	})
 
+	s.termsDischarger = &termsDischarger{}
+	termsDischarger := bakerytest.NewDischarger(nil, s.termsDischarger.thirdPartyChecker)
+
+	keyring := bakery.NewPublicKeyRing()
```

Comment:
If you use httpbakery.NewPublicKeyRing, you won't need to explicitly add the public keys, I think.